charts/spinnaker/values.yaml

```diff
@@ -146,6 +146,7 @@ dockerRegistries:
 - name: dockerhub
   address: index.docker.io
   noValidate: false
+  repositoriesRegex: '^filtered-repository$'
```

Comment:
Where will the value for `filtered-repository` be set?

Reply:
RIght [here](https://github.com/PicnicSupermarket/picnic-spinnaker/pull/959/files#diff-10df94f1c923d86a75c3e456c3c8f0b3e4e7dd896db2f6f885f56b872228e578R47)